client/src/components/Autocomplete/Autocomplete.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react';
+import Downshift from 'downshift';
+import { withStyles } from '@material-ui/core/styles';
+import MenuItem from '@material-ui/core/MenuItem';
+import { Wrapper, Menu, InputWrapper, Input, Button } from './Autocomplete.styled';
+import { colors } from 'style';
+
+const StyledMenuItem = withStyles(() => ({
+  root: {
+    fontFamily: 'Assistant',
```

Comment:
Can we create a new theme?

Reply:
I think we use MUI theme, so it's better if it goes there

Reply:
I added a default `fontFamily` to the `MuiTheme` so now it's not necessary.
I will open a separate pr to fixing the MuiTheme once this is merged.